### PR TITLE
CHK-131: Fix Mexico rules

### PR DESCRIPTION
**Pull request by garrucho**
The Mexico REGEX was missing one of their _nationalDestinationCodes_. The `Phone.validate(number)` was invalid for "988".

image

Also adds a test for all Mexico NDC's.

---

For test in a store:
- quick cart: https://mxsonyb2c.myvtex.com/checkout/cart/add/?sku=418&qty=1&seller=1&sc=1
- phone: (988) 957 7795

### CODE REVIEW

File: spec/countries/MEX-spec.coffee [hunk 1]
```diff
@@ -79,3 +79,24 @@ describe 'Mexico', ->
...
+
+			# Act
+			for ndc in Phone.countries['52'].nationalDestinationCode
+				if ndc.length is 2
+					number = prefix + ndc + suffix8
+				else
+					number = prefix + ndc + suffix
+
+				result = Phone.validate(number)
+				console.log(number, result)
```
Comment (marcoskwkm) [inline spec/countries/MEX-spec.coffee:97]: Left by mistake?

File: spec/countries/MEX-spec.coffee [hunk 2]
```diff
@@ -79,3 +79,24 @@ describe 'Mexico', ->
...
+
+				# Assert
+				if !result
+					console.log 'NDC missing: ', ndc
```
Comment (marcoskwkm) [inline spec/countries/MEX-spec.coffee:101]: Ditto

File: spec/countries/MEX-spec.coffee (hunk 1 above)
Comment (garrucho) [reply spec/countries/MEX-spec.coffee]: Ops, yes! Removed.

File: spec/countries/MEX-spec.coffee (hunk 2 above)
Comment (garrucho) [reply spec/countries/MEX-spec.coffee:100]: This is intended. The log will show missing NDCs along with the test results. 😄 
I've copied from this other test: https://github.com/vtex/front.phone/blob/master/spec/countries/NANP-spec.coffee#L14-L22

File: spec/countries/MEX-spec.coffee [hunk 3]
```diff
@@ -79,3 +79,23 @@ describe 'Mexico', ->
...
+		it 'all Mexico NDCs', ->
+			# Arrange
+			prefix  = "+52 "
+			suffix = " 111 2222"
+			suffix8 = "0 111 2222"
+
+			# Act
+			for ndc in Phone.countries['52'].nationalDestinationCode
+				if ndc.length is 2
+					number = prefix + ndc + suffix8
```
Comment (garrucho) [inline spec/countries/MEX-spec.coffee:92]: @marcoskwkm Your opinion: is it too ugly? Any better idea? hahaha

File: spec/countries/MEX-spec.coffee (hunk 2 above)
Comment (marcoskwkm) [reply spec/countries/MEX-spec.coffee:100]: Ok, makes sense.

File: spec/countries/MEX-spec.coffee (hunk 3 above)
Comment (marcoskwkm) [reply spec/countries/MEX-spec.coffee:92]: Looks fine to me.

**Pull request merged**